### ISSUE TITLE
[

### ISSUE THREAD

**Issue by jcsia87** [tomi/vscode-rf-language-server#51]

**Comment by jcsia87** [comment]
Let's say I have the following test case

Valid Login
    [Template]  Login With Username And Password
    validuser1  validpassword1
    validuser2  validpassword2

When I crtrl+click the template keyword Login With Username And Password, I'm not able to do so.

I can ctrl+click to go to the keyword definition when it's not a template. My settings.json looks like below

    "rfLanguageServer.includePaths": [
        "**/*.robot",
        "**/*.py"
    ],

**Comment by humbienri** [comment]
If you look at the know issues for this VSCode extension, I do believe that Templates are not yet supported.

**Comment by jcsia87** [comment]
@humbienri Oh, thanks for pointing that out.

**Comment by tomi** [comment]
@jcsia87 Yes, like @humbienri mentioned, support for `[Template]` has not been implemented yet.
